_data/sites.json

```diff
@@ -934,6 +934,15 @@
         ]
     },
 
+    {
+        "name": "Bitwarden",
+        "url": "https://bitwarden.com/help/article/delete-your-account",
+        "difficulty": "easy",
+        "domains": [
+            "bitwarden.com"
+        ]
+    },
+
```

Comment:
Can you move it to be above `Blinkist` to keep the list sorted? 

Thanks!

Reply:
Oops! I must've missed it.

Reply:
Don't worry, the CI didn't kick in so you really couldn't know.